main/zoe/api/zoe.md

```diff
@@ -200,14 +245,133 @@ either the `proposal `or `payments` are empty, indicate this by
 omitting that argument or passing `undefined`, instead of passing an
 empty record.
 
-The `proposal` has three parts: 
-- `want`: An object with keywords as keys and `amounts` as values.
-- `give`: An object with keywords as keys and `amounts` as values.
-- `exit`: Specifies the payout-liveness policy Zoe can guarantee for the offer.
+`exit`'s value should be an `exitRule`, an object with three possible keys for
+key:value pairs:
+- `waived`: The user can't cancel and relies entirely on the smart contract to promptly finish their offer. Takes a `null` value.
+- `onDemand`: The user can cancel on demand. Takes a `null` value.
+- `afterDeadline`: The offer is automatically cancelled after a deadline, as determined by its `timer` and `deadline` properties and
+  `deadline`'s value, which is denoted in number of seconds.
+
+
+```js
+const myProposal = harden({
+  give: { Asset: quatloos(4)},
+  want: { Price: moola(15) },
+  exit: { afterDeadline: {
+    timer,
+    deadline: 100,
+  }}
+})
+```
 
 `paymentKeywordRecord` is a record with keywords as keys, with
  values of the actual `payments` to be escrowed. A `payment` is
  expected for every entry under `give`.
  
  `offer()` returns a `promise` for a `userSeat`. 
- 
+```js
+const paymentKeywordRecord = { 
+  'Asset' : quatloosPayment, 
+  'Price' : moolaPayment 
+};
+```
+### UserSeat object
+
+Zoe uses `seats` to represent `offers`, and has two `seat` facets a `ZCFSeat` and a `UserSeat`.
+
+`Seats` represent active `offers` and let contracts and users interact with them. `ZCFSeats` 
+are used within contracts and with `zcf.` methods. `UserSeats` represent `offers` external to 
+Zoe and the contract. The party who exercises an `invitation` and sends the `offer()` message
+to Zoe gets a `UserSeat` that can check `payouts`' status or retrieve their `offer` result.
+The result is whatever the contract chooses to return. This varies, but examples
+are a `string` and an `invitation` for another user.
+
+Also, a `userSeat` can be handed to an agent outside Zoe and the contract, letting
+them query or monitor the current state, access the `payouts` and result,
+and, if it's allowed for this `seat`, call `tryExit()`. 
+
+Since you can potentially exit the `seat` if you have a reference to it, you should
+only share your `UserSeat` with very trusted parties.
```

Comment:
```suggestion
Since anyone can attempt to exit the `seat` if they have a reference to it,
you should only share a `UserSeat` with trusted parties.
```

"trusted" parties is a technical term for us. No need for "very trusted".